docs/machine-learning/tutorials/sentiment-analysis.md

```diff
@@ -343,11 +343,7 @@ Create a header for the predictions using the following code:
 
 [!code-csharp[OutputHeaders](~/samples/machine-learning/tutorials/SentimentAnalysis/Program.cs#AddInfoMessage "Display prediction outputs")]
 
-Before displaying the predicted results, combine the original comment with its predicted sentiment using the [Zip()](xref:System.Linq.Enumerable.Zip%2A) method:
-
-[!code-csharp[BuildTuples](~/samples/machine-learning/tutorials/SentimentAnalysis/Program.cs#BuildSentimentPredictionPairs "Build the pairs of sentiment data and predictions")]
-
-Now that `SentimentText` and `Sentiment` are combined in a class, display the results:
+Because `SentimentPrediction` inherited from `SentimentData`, the `Transform()` method populated `SentimentText` with the predicted fields. As the ML.NET process processes, each component adds columns, and this makes it easy to display the results:
```

Comment:
Suggestion:

"Because `SentimentPrediction` is inherited from `SentimentData`, the `Transform()` method populated `SentimentText` with the values in the `SentimentText` column. Throughout the ML.NET model building process, each component adds columns, and this makes it easy to display the result"

Reply:
Excellent! Thanks, @luisquintanilla!

Reply:
It's not clear how the concepts in the two sentences connect here. What does the user need to know? That the fields in SentimentPrediction will be in the transformed result. The inheritance is an implementation detail.

Reply:
But the fields need to be there in order for the results to appear. Modified. Any clearer?